projects/canopy/src/styles/utils.scss

```diff
@@ -7,3 +7,18 @@
 .lg-unstyled-link {
   @include lg-unstyled-link;
 }
+
+.lg-footer-buttons {
```

Comment:
I wonder does this need a class ? I feel like the standard behaviour would always be to have this margin when two buttons or links are adjacent to one another.

There may be some exceptions in existing components but we could probably find them..
https://deploy-preview-416--legal-and-general-canopy.netlify.app/?path=/story/components-form-input--with-multiple-button-suffixes

Reply:
Do you mean to put this in the buttons styles?

Reply:
I thought about adding it to the button component but I feel like it's not the right thing to do. The button component handles the style for a single button and if we start adding all the different handling of multiple buttons it might get bloated.

Reply:
Discussing this with @andy-polhill he suggested to have a `lg-button-group` component so that we don't have to manually add the class and it's more built into the API.

Reply:
I wonder if it could be a simple component..
```html
<lg-button-group>
  <lg-button />
  <lg-button />
</lg-button-group>
```

My only concern with the class is that it isn't something we have used much before, a developer in an application would have to step out of the normal Canopy paradigms to add that class. There is also a risk they may not know it exists.

Reply:
Done 👍